### PR TITLE
Make error output of tap-min more verbose

### PR DESCRIPTION
**Pull request by ghost**
Additionally update coloring for better readability.

### CODE REVIEW

**Comment by ghost** [comment]
Let me know if the PR is alright or needs some more minor changes, thanks!

File: index.js [hunk 1]
```diff
@@ -32,12 +32,22 @@ const reporter = () => {
 
     output.write('\n' + chalk.white(assert.name) + '\n')
     if (assert.diag) {
-      const d = assert.diag
-      output.write(chalk.gray(
-        `\toperator: ${d.operator}\n` +
-        `\texpected: ${util.inspect(d.expected)}\n` +
-        `\tactual:   ${util.inspect(d.actual)}\n`
-      ))
+        const d = assert.diag
```
Comment (derhuerst) [inline index.js]: The indentation does not match the rest of the file. (Which in turn doesn't match the `.editorconfig`. 😊)

**Comment by ghost** [comment]
@derhuerst Amended the commit to two spaces now, apologies for the oversight! Don't hesitate to bug me if there's still something wrong with it 👉!

**Comment by derhuerst** [comment]
Looks good, thanks!

Is this a breaking change? Nobody should programmatically depend on the visual output of `tap-min`, so my gut feeling is publishing this is a feature release.

**Comment by ghost** [comment]
I would agree that this is a feature release, like you said yourself, programmatically depending on the output of `tap-min` would be kind of odd and that would be the only instance I could think of where this would qualify as a breaking change.

**Comment by derhuerst** [comment]
Using which name and optionally eMail do you want to be mentioned as a contributor?

Example: https://github.com/derhuerst/tap-min/blob/348167118944083da4c94a2e53b71aad50462408/package.json#L26

**Comment by ghost** [comment]
`Tim Eulitz <tim.eulitz@wikimedia.de>`

Thanks for maintaining!

**Pull request merged**